train.py

```diff
@@ -260,13 +260,33 @@ def compute_metrics():
                 logits = model(X, Y)
                 loss = raw_model.last_loss
             losses[k] = loss.item()
+        # TODO @Markus you mentioned this was computing metrics wrongly, so I commented it out, if this is
```

Comment:
This computes the _wrong_ metrics, but the computation is correct. We should add more tests so we're comfy with the metrics.